interface/assets/css/project.css

```diff
@@ -3,3 +3,5 @@
   padding: 2px;
   width: auto;
 }
+
+.top-buffer { margin-top:20px; }
```

Comment:
I wonder if this should follow the existing style to some degree?

Reply:
I'm not sure what you mean? Currently, the main content of a page would be overlapped at the top to some degree by the navbar. Adding some margin to the content container in `base.html` should solve that.

Reply:
Existing CSS code/indentation/spacing style, not the final result.. sorry for the confusion!

Reply:
Oh I see! Yeah, that would be way better :)